content/patterns/adapt-to-content-and-context.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Adapt to content and context
+
+category: Understanding and influencing decisions
+
+weight: 1
+
+archived: false
+
+archive_reason:
+
+future_pattern: false
+
+future_pattern_reason:
+
+images:
+ - url: /images/adapt-to-content-and-context.svg
+alt: Anomalies are detected in what someone says and their case is automatically flagged as urgent
+
+advantages:
+ - Analysing language-based signals has been proven to effectively prioritise vulnerable people’s access to services 
+ - Contextual cues such as location, time of day and others can be used to protect people from harm
+ - People can use their own language to communicate with services in an accessible and inclusive way
+ - Established guidance for natural language processing and automated decision making ensures experiences can be consistent in regulated sectors
+
+
+limitations:
+ - Without guardrails there is room for misinterpretation of the words people use or introduction of bias
+ - False positives may occur which negatively impact user experience and trust
+ - Inferring vulnerability or mental state by analysing user interactions needs to be handled carefully, particularly in regulated sectors, and the resulting information should not be used for any other purposes
+
+
+examples:
+ - title: Salesforce
+   url: https://jmcloudservices.com/blog/enhancing-case-classification-and-routing-with-tone-and-sentiment-analysis-using-salesforce-einstein-generative-ai-apex-and-flow-part-1
+   description: Analyses tone and sentiment to automatically triage customer service calls
+
+ - title: Lime
+   url: https://micromobilitybiz.com/lime-unveils-new-technology-to-cut-down-on-drunk-and-pavement-riding/
+   description: Makes late-night riders take a reaction test, aimed at deterring drunk driving
+
+ - title: Monzo
```

Comment:
I love this monzo pattern, have been wanting to get it into the catalogue for ages